infra/gcp/ensure-test-storage.sh

```diff
@@ -0,0 +1,27 @@
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# The service account name for the image promoter test account
+PROMOTER_TEST_SVC="k8s-infra-gcr-promoter-test"
+
+# GCP project names.
+STAGING_PROJECT="k8s-staging-cip-test"
+PROD_PROJECT="k8s-cip-test-prod"
+
+ALL_PROJECTS=("${TEST_PROJECT}")
```

Comment:
Shouldn't this be `${STAGING_PROJECT} ${PROD_PROJECT}`? (I think `TEST_PROJECT` is undefined)

Reply:
good catch!